packages/systemd-257/systemd-257.spec

```
@@ -0,0 +1,783 @@
+%exclude %{_cross_unitdir}/capsule@.service
+
+# Soft reboot of userspace is not implemented and supported
+%exclude %{_cross_unitdir}/soft-reboot.target
+%exclude %{_cross_unitdir}/systemd-soft-reboot.service
+
+# Disable auto load of credentials via udevadm
+%exclude %{_cross_unitdir}/systemd-udev-load-credentials.service
+
+# Disable the notifier service: systemd-networkd-persistent-storage.service.
+# Without this service Bottlerocket loses support for reusing DHCP leases on
+# reboot but avoids setting up a storage dependency for systemd-networkd
+%exclude %{_cross_unitdir}/systemd-networkd-persistent-storage.service
+
+# Exclude target used by ssh services
+%exclude %{_cross_unitdir}/ssh-access.target
+
+# Exclude desktop related targets.
+%exclude %{_cross_unitdir}/bluetooth.target
+%exclude %{_cross_unitdir}/printer.target
+%exclude %{_cross_unitdir}/smartcard.target
+%exclude %{_cross_unitdir}/sound.target
+%exclude %{_cross_unitdir}/usb-gadget.target
+
+# Exclude remote filesystem targets.
+%exclude %{_cross_unitdir}/remote-fs-pre.target
+%exclude %{_cross_unitdir}/remote-fs.target
+
+# Exclude user-related functionality.
+%exclude %{_cross_unitdir}/user-runtime-dir@.service
+%exclude %{_cross_unitdir}/user@.service
+%exclude %{_cross_unitdir}/user@.service.d
+%exclude %{_cross_unitdir}/user@0.service.d
+%exclude %{_cross_unitdir}/user-.slice.d/10-defaults.conf
+%exclude %{_cross_unitdir}/user.slice
+%exclude %{_cross_userunitdir}
+%exclude %{_cross_libdir}/systemd/systemd-user-runtime-dir
+%exclude %{_cross_libdir}/systemd/user-preset/90-systemd.preset
+
+# Exclude units related to the initrd.
+%exclude %{_cross_unitdir}/initrd-root-fs.target.wants
+
+# Exclude repart service since we have custom repart logic.
+%exclude %{_cross_unitdir}/systemd-repart.service
+
+# Exclude upstream binfmt functionality.
+%exclude %{_cross_libdir}/systemd/systemd-binfmt
+%exclude %{_cross_unitdir}/systemd-binfmt.service
+%exclude %{_cross_unitdir}/proc-sys-fs-binfmt_misc.automount
+%exclude %{_cross_unitdir}/sysinit.target.wants/proc-sys-fs-binfmt_misc.automount
+%exclude %{_cross_unitdir}/sysinit.target.wants/systemd-binfmt.service
+
+# Exclude functionality related to offline updates.
+%exclude %{_cross_libdir}/systemd/systemd-update-done
+%exclude %{_cross_libdir}/systemd/systemd-update-helper
+%exclude %{_cross_systemdgeneratordir}/systemd-system-update-generator
+%exclude %{_cross_unitdir}/sysinit.target.wants/systemd-update-done.service
+%exclude %{_cross_unitdir}/system-update-cleanup.service
+%exclude %{_cross_unitdir}/system-update-pre.target
+%exclude %{_cross_unitdir}/system-update.target
+%exclude %{_cross_unitdir}/systemd-update-done.service
+
+%dir %{_cross_libdir}/udev
+%{_cross_libdir}/udev/ata_id
+%{_cross_libdir}/udev/cdrom_id
+%{_cross_libdir}/udev/dmi_memory_id
+%{_cross_libdir}/udev/fido_id
+%{_cross_libdir}/udev/mtd_probe
+%{_cross_libdir}/udev/scsi_id
+%exclude %{_cross_libdir}/udev/v4l_id
+%{_cross_libdir}/udev/iocost
```

nit: includes come before excludes
```suggestion
%{_cross_libdir}/udev/iocost
%exclude %{_cross_libdir}/udev/v4l_id
```